frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/useDeleteLayout.ts

```diff
@@ -6,6 +6,6 @@ import { useAppContext } from '../../../hooks/useAppContext';
 export const useDeleteLayout = () => {
   const { org, app } = useStudioUrlParams();
   const { selectedLayoutSet } = useAppContext();
-  const { mutate: deleteLayout } = useDeleteLayoutMutation(org, app, selectedLayoutSet);
-  return useMemo(() => deleteLayout, [deleteLayout]);
+  const { mutate: deleteLayout, isPending } = useDeleteLayoutMutation(org, app, selectedLayoutSet);
+  return useMemo(() => ({ deleteLayout, isPending }), [deleteLayout, isPending]);
```

Comment:
Du kan godt bare returnere resultatet av `useDeleteLayoutMutation` direkte her. Da får vi med alle variablene fra React Query og gjør det mer skalerbart.
```suggestion
  return useDeleteLayoutMutation(org, app, selectedLayoutSet);
```